package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@piwik/referrer-spam-blacklist",
+  "version": "0.0.0",
```

Comment:
is it normal that version is set to 0.0.0? ideally, we wouldn't specify a version if possible, as the package is so far version'ed using Github releases: https://github.com/piwik/referrer-spam-blacklist/releases